### ISSUE TITLE
Error while cloning a Template

### ISSUE THREAD

**Issue by sanjay1839** [vmware/pyvmomi-community-samples#478]
Getting this error while clone a template to create a vm, could not find what the error is can you please help

cloning VM...
there was an error

**Comment by prziborowski** [comment]
Could you post a snippet of the code you are using?
Typically the API don't print "there was an error".

**Comment by prziborowski** [comment]
```
task = template.Clone(folder=destfolder, name=vm_name, spec=clonespec)
wait_for_task(task)
```

Can you change this to (add this line toward the top with the other imports):
```
from pyVim.task import WaitForTask
```
And the template.Clone call:
```
task = template.Clone(folder=destfolder, name=vm_name, spec=clonespec)
WaitForTask(task)
```

And then report what the error is?
I think that will include the actual fault returned by vCenter rather than a more generic print statement.

**Comment by sanjay1839** [comment]
Sure i will do that

**Comment by prziborowski** [comment]
OK, that is helpful.
It seems that the spec.pool that is provided by the script has some problem.
Are you providing the --cluster-name option to the script? If yes, does that cluster have hosts?
If you aren't providing it, do you have any clusters in your vCenter that don't have hosts (the script might be picking one of those by default)?

**Comment by sanjay1839** [comment]
Yes i am  giving the cluster name and the cluster has the host added to it and the template is in that host and i have tried deleting the cluster which doesn't have host got the same error.

can you give me the proper command to run
